src/main/java/org/kohsuke/github/GHContentUpdateResponse.java

```diff
@@ -21,7 +25,101 @@ public GHContent getContent() {
      *
      * @return the commit
      */
-    public GHCommit getCommit() {
+    public Commit getCommit() {
```

Comment:
Regardless of other considerations, this is breaking change.  You can work around this with BridgeMethods, but it will still need to be done differently.  (I understand this deeply broken due to lack of testing, to the point that it is virtually impossible that anyone is actually using this API as it stands, but we can't take API breaking change at this time.)

Reply:
Do you mean we should add another method or we should copy data to `GHCommit`?

Reply:
Given that has returns null and [this has been this way forever](https://github.com/hub4j/github-api/commit/387d4e5bc976f97d3d6c2cbedaa5dba513a9fd70), we just need to have a bridge method that returns `null` if someone calls the `GHCommit getCommit()` method.

Reply:
I tried to add a bridge method for `GHCommit`. Is this correct?